include/sys/dsl_bookmark.h

```diff
@@ -36,8 +36,25 @@ typedef struct zfs_bookmark_phys {
 	uint64_t zbm_guid;		/* guid of bookmarked dataset */
 	uint64_t zbm_creation_txg;	/* birth transaction group */
 	uint64_t zbm_creation_time;	/* bookmark creation time */
+
+	/* the following fields are reserved for redacted send / recv */
+	uint64_t zbm_redaction_obj;	/* redaction list object */
+	uint64_t zbm_flags;		/* ZBM_FLAG_* */
+	uint64_t zbm_referenced_bytes_refd;
+	uint64_t zbm_compressed_bytes_refd;
+	uint64_t zbm_uncompressed_bytes_refd;
+	uint64_t zbm_referenced_freed_before_next_snap;
+	uint64_t zbm_compressed_freed_before_next_snap;
+	uint64_t zbm_uncompressed_freed_before_next_snap;
```

Comment:
Can we confirm these are the finalized fields needed for redacted send / recv.

Reply:
These were provided to me by @ahrens and confirmed by @pcd1193182. @pcd1193182 can you just confirm that this is correct?

Reply:
Those are correct.